packages/xo-web/src/common/render-xo-item.js

```diff
@@ -77,15 +77,24 @@ export const Host = decorate([
       ),
     }
   }),
-  ({ host, pool, link, newTab }) => {
+  ({ host, pool, link, newTab, memoryFree }) => {
     if (host === undefined) {
       return UNKNOWN_ITEM
     }
 
     return (
       <LinkWrapper link={link} newTab={newTab} to={`/hosts/${host.id}`}>
         <Icon icon='host' /> {host.name_label}
-        {pool !== undefined && ` (${pool.name_label})`}
+        {memoryFree && (
+          <span>
+            {' ('}
+            {_('freeSpace', {
+              freeSpace: formatSize(host.memory.size - host.memory.usage),
```

Comment:
Actually, this should say "... GiB RAM free". Sorry, I didn't realize it could be confusing before testing it.